submissions/Test123423235/CarRacingGame/Renderer.java

```diff
@@ -0,0 +1,115 @@
+package CarRacingGame;
+
+public class Renderer {
+    //Attributes
+    private IRenderObject[] Store = new IRenderObject[0];
```

Comment:
In Java, attributes should start with a lowercase. -> `store` otherwise it is easily confused with a class. This is not wrong just a convention.